### PR TITLE
Add wms legend support tests

### PR DESCRIPTION
**Pull request by theduckylittle**
Added a test case to ensure coverage of the OGC:WMS type + legend url code.

### CODE REVIEW

**Comment by ingenieroariel** [comment]
@JorgeMartinezG can you help suggest the changes to make the tests on Python 3.5 and Elastic 5.0 work?

I think it is just related to how many items there are in the db (4 vs 5) in one test and how time is formatted in the new example.

**Comment by JorgeMartinezG** [comment]
@theduckylittle @ingenieroariel 

https://github.com/theduckylittle/registry/blob/ee2121ccebf3b5cf5a93abb047df34f51267c6f3/test_registry.py#L379
The assertion must be changed to 5 because a new layer is added into the tests.

https://github.com/theduckylittle/registry/blob/ee2121ccebf3b5cf5a93abb047df34f51267c6f3/test_registry.py#L689
Is this line necessary? If yes, print in python 3 must be written as a function

**Comment by theduckylittle** [comment]
@JorgeMartinezG  - I'm not sure how the debugging statement I put in there got through, I'm usually not so sloppy, sorry!  The counts being off puzzles me too as, locally, I thought I had all the tests running fine! 

Both changes have been made and pushed.

**Comment by JorgeMartinezG** [comment]
@theduckylittle  There is a conditionat to skip heatmap tests if elasticsearch version is not 5.1

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 class PyTest(TestCommand):
     def finalize_options(self):
         TestCommand.finalize_options(self)
-        self.test_args = []
+        self.test_args = ["--cov","registry","--cov-report","term-missing"]
```
Comment (ingenieroariel) [inline setup.py]: Travis is apparently not liking these arguments.

Comment (ingenieroariel) [inline setup.py]: We usually run tests with coverage like:
```coverage run --source=registry setup.py test```

**Pull request merged**